@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/targeting/fn_targetingVariable.sqf

```diff
@@ -25,27 +25,24 @@ private _visibleTargets = [];
 //TSD ATK LIST
 if (([_heli, 1] call fza_fnc_mpdGetCurrentDisplay == "tsd") && (_heli getVariable "fza_ah64_tsdmode" == "atk") && !(cameraView == "GUNNER")) then {
 	fza_ah64_tsddisptargs = fza_ah64_targetlist select {
```

Comment:
For now can we just make it so that it displays whatever is in the `targetlist`? Right now we are having issues with things not appearing when they should (fcr picked up target (heli in ground mode) but wasn't visible)

This can be a crutch until we can properly do the targeting stuff.

Reply:
so get rid of select and make
fza_ah64_tsddisptargs = fza_ah64_targetlist?